DAOS-Support.md

```diff
@@ -2,8 +2,7 @@
 
 [DAOS](https://github.com/daos-stack/daos) is supported as a backend storage system in dcp. The build instructions for
 enabling DAOS support can be found here:
-[Enable DAOS](https://github.com/hpc/mpifileutils/doc/rst/build.rst).
-
+[Enable DAOS](https://mpifileutils.readthedocs.io/en/latest/build.html).
```

Comment:
```suggestion
[Enable DAOS](https://mpifileutils.readthedocs.io/en/latest/build.html#build-everything-directly).
```
@dsikich Do you think we should navigate specifically to "Build Everything Directly", since that is the section with DAOS instructions? I'm not sure if Spack works for compiling DAOS (I remember having some trouble with it)

Reply:
But I really don't think this is a big deal; the link you have is good.